test/Common/TestOrchestratorClient.cs

```diff
@@ -54,11 +54,15 @@ public async Task<DurableOrchestrationStatus> GetStatusAsync(bool showHistory =
             return status;
         }
 
+        public async Task RaiseEventAsync(string eventName) => await innerClient.RaiseEventAsync(instanceId: instanceId, eventName: eventName);
+
         public async Task RaiseEventAsync(string eventName, object eventData)
         {
             await this.innerClient.RaiseEventAsync(this.instanceId, eventName, eventData);
         }
 
+        public async Task RaiseEventAsync(string taskHubName, string instanceid, string eventName, string connectionName = null) => await innerClient.RaiseEventAsync(taskHubName: taskHubName, instanceId: instanceid, eventName: eventName);
```

Comment:
Would this be a breaking change? For example, if customers previously had code like the following:
```csharp
client.RaiseEventAsync(instanceId, eventName, "string event data");
```
Will that now conflict with the new overload? Your use of parameter names scares me. If so, we should consider reducing the scope of your change - i.e. delete the new `RaiseEventAsync(string, string, string, string = null)`

Reply:
Yeah like i put in the PR comment, one has to now get extra-specific when they want to use either of these methods in order to disambiguate between what we have today and the proposed change here.

I think you might be right that the requirement for people to do this is indeed a breaking change. Shoot.